src/excel/NamedItemContainer.js

```diff
@@ -50,7 +50,7 @@ export class NamedItemContainer {
       });
     } catch (e) {
       if ((e.details && e.details.code === 'ItemAlreadyExists') && e.statusCode !== 409) {
-        throw new StatusCodeError(e.message, 409, e.code, e);
+        throw new StatusCodeError(e.message, 409);
```

Comment:
this was actually badly constructed (there was never a 4th arg to the `StatusCodeError` constructor), but still the test succeeded